build.gradle

```diff
@@ -112,10 +112,14 @@ test {
 intellij {
     version = ideaVersion //for a full list of Jetbrains IDEA releases please see https://www.jetbrains.com/intellij-repository/releases
     pluginName = 'Kubernetes by Red Hat'
-    plugins = ['yaml']
+    plugins = ['yaml', 'com.redhat.devtools.intellij.telemetry:0.0.2.18' ]
     updateSinceUntilBuild = false
 }
 
+runIde {
```

Comment:
The same should be added for integration tests see https://github.com/redhat-developer/intellij-quarkus/blob/main/build.gradle#L151

Reply:
done